features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Monetization/index.jsx

```diff
@@ -207,76 +208,102 @@ class Monetization extends Component {
             return <Progress />;
         }
         return (
-            <Grid item xs={6}>
-                <Typography variant='h4' gutterBottom>
-                    <FormattedMessage id='Apis.Details.Monetization.Index.monetization' defaultMessage='Monetization' />
-                </Typography>
+            <Grid>
```

Comment:
You can get rid of this tag.

Reply:
fixed in d1ec43ed4f29d7fc678637fc70f1a68f9e8afca9